src/cloudWatchLogs/document/logStreamDocumentProvider.ts

```diff
@@ -45,10 +45,12 @@ export class LogStreamDocumentProvider implements vscode.TextDocumentContentProv
         position: vscode.Position,
         token: vscode.CancellationToken
     ): Promise<vscode.Definition | vscode.LocationLink[] | undefined> {
+        let result: telemetry.Result
         const activeUri = document.uri
         const logGroupInfo = parseCloudWatchLogsUri(activeUri).logGroupInfo
         if (logGroupInfo.streamName) {
             // This means we have a stream file not a log search.
+            // TODO: Should there be any telemetry output here?
             return
```

Comment:
Don't think telemetry is needed here. 

Different question: is this an expected situation? If it signals a logic bug or bad input, that should be logged, if not raise an error.

Reply:
Right now we are still missing a `glob` pattern to distinguish the logStream uri from a logGroup uri when we register the `defintionProvider`. So this case is hit anytime someone attempts to look for a definition in a single logStream document.